include/PianoRoll.h

```diff
@@ -388,6 +390,7 @@ protected slots:
 	EditModes m_ctrlMode; // mode they were in before they hit ctrl
 
 	bool m_mouseDownRight; //true if right click is being held down
+	bool m_markFromMenu; // true if from right-click menu
```

Comment:
As far as I can tell this is just an indirect way to pass a parameter to `markSemitone`. I would strongly prefer to avoid this method. Couldn't you just add a `fromMenu` parameter to `markSemitone` that defaults to true? If leaving existing calls unmodified is your concern, this should solve the issue.

Reply:
I can add a default parameter. 👍